src/content/en/2021/resource-hints.md

```diff
@@ -1,20 +1,609 @@
+        <td class="numeric">27.0%</td>
+      </tr>
+      <tr>
+        <td>`fonts.googleapis.com`</td>
+        <td class="numeric">14.1%</td>
+        <td class="numeric">4.6%</td>
+        <td class="numeric">3.1%</td>
+        <td class="numeric">21.7%</td>
+      </tr>
+      <tr>
+        <td>`s.w.org`</td>
+        <td class="numeric">20.0%</td>
+        <td class="numeric">0.3%</td>
+        <td class="numeric">-</td>
+        <td class="numeric">20.3%</td>
+      </tr>
+      <tr>
+        <td>`cdn.shopify.com`</td>
+        <td class="numeric">-</td>
+        <td class="numeric">1.8%</td>
+        <td class="numeric">12.2%</td>
+        <td class="numeric">14.0%</td>
+      </tr>
+      <tr>
+        <td>`www.google-analytics.com`</td>
+        <td class="numeric">1.3%</td>
+        <td class="numeric">4.1%</td>
+        <td class="numeric">0.2%</td>
+        <td class="numeric">5.5%</td>
+      </tr>
+      <tr>
+        <td>`www.googletagmanager.com`</td>
+        <td class="numeric">1.9%</td>
+        <td class="numeric">3.1%</td>
+        <td class="numeric">0.3%</td>
+        <td class="numeric">5.3%</td>
+      </tr>
+      <tr>
+        <td>`siteassets.parastorage.com`</td>
+        <td class="numeric">-</td>
+        <td class="numeric">-</td>
+        <td class="numeric">5.2%</td>
+        <td class="numeric">5.2%</td>
+      </tr>
+      <tr>
+        <td>`images.squarespace-cdn.com`</td>
+        <td class="numeric">-</td>
+        <td class="numeric">4.5%</td>
+        <td class="numeric">-</td>
+        <td class="numeric">4.5%</td>
+      </tr>
+      <tr>
+        <td>`ajax.googleapis.com`</td>
+        <td class="numeric">2.3%</td>
+        <td class="numeric">1.8%</td>
+        <td class="numeric">0.4%</td>
+        <td class="numeric">4.5%</td>
+      </tr>
+    </tbody>
+  </table>
+
+<figcaption>{{ figure_link(caption="Most popular third parties connected to using resource hints.", sheets_gid="1254656281", sql_file="preload_host_by_url.sql") }}</figcaption>
+</figure>
+
+Analyzing the table above, 42.7% of all pages which include a `preload` hint are preloading resources hosted on adservice.google.com, while s.w.org (WordPress) is the most popular domain for `dns-prefetch` and `fonts.gstatic.com` for the `preconnect` directive.
+
+{{ figure_markup(
+  image="google-fonts.png",
+  alt="Google Fonts instructions to embed a font.",
+  caption='Google Fonts instructions to preconnect to fonts.gstatic.com and fonts.googleapis.com. From <a hreflang="en" href="https://fonts.google.com/">Google Fonts</a>.',
+  description='Code snippet titled "Use on the web" and subtitled with "To embed a font, copy the code into the `<head>` of your html". Code snippet states `<link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">`',
+  width=536,
+  height=510
```

Comment:
This figure appears very zoomed in. Should the dimensions be halved?

Reply:
Should I resize the original image or change the width/height attributes to preserve the higher-resolution image?

Reply:
just change height and width and leave image itself alone. In same way we say charts are 600px when they are actually 1200px.